container/container.go

```diff
@@ -183,6 +185,34 @@ func (container *Container) WriteHostConfig() error {
 	return json.NewEncoder(f).Encode(&container.HostConfig)
 }
 
+// SetupWorkingDirectory sets up the container's working directory as set in container.Config.WorkingDir
+func (container *Container) SetupWorkingDirectory() error {
+	if container.Config.WorkingDir == "" {
+		return nil
+	}
+	container.Config.WorkingDir = filepath.Clean(container.Config.WorkingDir)
+
+	pth, err := container.GetResourcePath(container.Config.WorkingDir)
+	if err != nil {
+		return err
+	}
+
+	pthInfo, err := os.Stat(pth)
+	if err != nil {
```

Comment:
Hm I just noticed this code was already there, only moved. Still wondering why it's needed